vortex-rs/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "vortex-rs"
+description = "Vortex with all builtin codecs and a sampling compressor."
```

Comment:
Do we want this crate to re-export all the underlying dependencies?

Reply:
We should probably also move a bunch of the docs in `vortex-array` here, but that can be a separate PR